addons/website/views/website_templates.xml

```diff
@@ -2418,6 +2464,69 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
     </tr>
 </template>
 
+<template id="list_hybrid" name="Any Search Results">
+    <t t-call="website.layout">
+        <div id="wrap">
+            <div class="container pt24 pb24" id="list_website_pages">
+                <t t-call="website.website_search_box_input">
+                    <t t-set="_classes" t-valuef="mt8 float-right"/>
```

Comment:
I think because of "float-right" here, the `".dropdown-menu"` overflows your search input:

![img-ii](https://user-images.githubusercontent.com/25032876/114899058-f0b62180-9e12-11eb-8628-6106d6dd7e74.png)